hyperspy/io.py

```diff
@@ -478,6 +478,9 @@ def save(filename, signal, overwrite=None, **kwds):
         ensure_directory(filename)
         if overwrite is None:
             overwrite = overwrite_method(filename)
+        if overwrite is False:
```

Comment:
@hentr yes, essentially, but not just `pass`. At the very least you should `_logger.warning(f"file {filename} exists, did not overwrite")`, or similar/better. We try to avoid silently passing things.